### PR TITLE
Feature/mosi add a new file mosi.html

### PR DESCRIPTION
**Pull request by mosimosu**
- 新增mosi.html

### CODE REVIEW

File: mosi.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <p>Hello, world.</p>
+  </body>
+</html>
```
Comment (Eshiunm) [inline mosi.html:11]: 讚讚

File: mosi.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <p>Hello, world.</p>
+  </body>
```
Comment (penspulse326) [inline mosi.html:10]: Good !!!!!!!!!!!

File: mosi.html (hunk 1 above)
Comment (Levia6603) [reply mosi.html:11]: 讚呦

File: mosi.html (hunk 2 above)
Comment (mosimosu) [inline mosi.html:10]: good!

**Pull request merged**